nilearn/decomposition/multi_pca.py

```diff
@@ -188,6 +196,20 @@ def __init__(self, n_components=20, smoothing_fwhm=None, mask=None,
         self.target_affine = target_affine
         self.target_shape = target_shape
         self.standardize = standardize
+        self.incremental_group_pca = incremental_group_pca
+
+        if incremental_group_pca:
+            try:
+                from sklearn.decomposition.incremental_pca import IncrementalPCA
+            except ImportError as exc:
```

Comment:
How about a simple warning instead of an exception in IncrementalPCA should not be available?